ANMN/LTSP/HourlyAggregator/aggregated_timeseries.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import print_function
```

Comment:
Please remove this file from this PR. This one should be _only_ about the first product.

Reply:
done.